src/LinearSolve.jl

```diff
@@ -38,12 +39,13 @@ function __init__()
   else
     IS_OPENBLAS[] = occursin("openblas", BLAS.get_config().loaded_libs[1].libname)
   end
+
+  @require Pardiso="46dd5b70-b6fb-5a00-ae2d-e8fea33afaf2" PardisoJL
```

Comment:
needs to be an include. See the DiffEqBase requires handling as an example.

Reply:
missed it sorry. fixed now